### DISCUSSION TITLE
Class "Filament\Support\Assets\AlpineComponent" not found

### DISCUSSION THREAD

**Discussion by arslonbeksaidov** [awcodes/filament-tiptap-editor#305] [Q&A]
In FilamentTiptapEditorServiceProvider.php line 36:
                                                             
  Class "Filament\Support\Assets\AlpineComponent" not found  
                                                             

Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1

**Comment by awcodes** [comment]
Sounds like you're not on Filament v3.
